src/app/app.html

```diff
@@ -1,14 +1,17 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title>
+          PVTrAck
```

Comment:
I changed the aria-label of title to "Peeveetrack". But it isn't read, as far as I know. I changed it to something outrageous just to see what happens, but I didn't hear the screenreader say it when I was navigating the page.